src/test/java/com/ksm/wstbackoffice/endpoint/EndPoint.java

```diff
@@ -0,0 +1,13 @@
+package com.ksm.wstbackoffice.endpoint;
+
+public final class EndPoint {
+    public static final int port = 8081;
```

Comment:
you specify port here and in the application-test.yml. Why do you need it?

Reply:
removed from application-test.yml because port=0 equalled 0 in constructor or method with annotation @BeforeAll